scripts/test/js_test.sh

```diff
@@ -7,6 +7,9 @@ then
 elif [[ ! -z "$CODECOV" ]]
 then
     export CMD="node ./node_modules/nyc/bin/nyc.js --reporter=lcovonly -R spec mocha"
+elif [[ ! -z "$WATCH" ]]
+then
+    export CMD="node ./node_modules/.bin/mocha --watch"
```

Comment:
`.bin` is just a directory for symlinks to executables, anything in `${package}/bin` will get symlinked there. if we want it to be consistent I can change it, I forgot that we usually specify the full path.